package.json

```diff
@@ -1,109 +1,35 @@
 {
-  "name": "@ubiquity/dollar",
-  "license": "MIT",
-  "scripts": {
-    "start": "run-s install:env install:contracts build:contracts run",
-    "prepare": "husky install",
-    "__test:watch:ci": "nodemon",
-    "build:ci": "run-s install:env install:contracts build:contracts build:next",
-    "install:env": "cp .env.example .env",
-    "install:contracts": "run-s _contracts:install:*",
-    "_contracts:install": "run-p _contracts:install:*",
-    "_contracts:install:dollar": "cd ./contracts/dollar && yarn",
-    "_contracts:install:nft": "cd ./contracts/ubiquistick && yarn",
-    "build:contracts": "run-s _contracts:build:contracts _contracts:build:artifacts _contracts:build:typechain _contracts:build:summary",
-    "_contracts:build:typechain": "typechain --target=ethers-v5 --out-dir ./fixtures/abi/types ./fixtures/abi/*.json",
-    "_contracts:build:contracts": "run-p _contracts:build:contracts:*",
-    "_contracts:build:contracts:nft": "cd ./contracts/ubiquistick && yarn build",
-    "_contracts:build:contracts:dollar": "cd ./contracts/dollar && yarn build",
-    "_contracts:build:artifacts": "run-p _contracts:build:artifacts:*",
-    "_contracts:build:artifacts:nft": "sh ./utils/build-artifacts.sh ./contracts/ubiquistick",
-    "_contracts:build:artifacts:dollar": "sh ./utils/build-artifacts.sh ./contracts/dollar",
-    "_contracts:build:summary": "node ./utils/generate-contracts-addresses.js",
-    "test:all": "run-s _contracts:test:*",
-    "_contracts:test:dollar": "cd ./contracts/dollar && yarn test",
-    "lint:all": "run-s _contracts:lint:*",
-    "_contracts:lint:dollar": "cd ./contracts/dollar && yarn lint",
-    "format": "run-p _format:*",
-    "_format:prettier": "prettier --write .",
-    "_format:eslint": "eslint . --ext .ts,.tsx --cache --fix --plugin tsc --rule 'tsc/config: [2, {configFile: \"./tsconfig.json\"}]'",
-    "build:next": "next build && next export --outdir dist",
-    "build": "run-s install:env install:contracts",
-    "run": "run-p run:*",
-    "run:contracts": "./utils/dev-chain-init.sh",
-    "run:ui": "next dev"
-  },
-  "_scripts": {
-    "start:headless": "run-s install:env install:contracts dev:contracts",
-    "build:ui": "yarn next build",
-    "_contracts:clean": "run-p _contracts:clean:*",
-    "_contracts:clean:dollar": "cd ./contracts/dollar && yarn clean",
-    "_contracts:clean:nft": "cd ./contracts/ubiquistick && yarn clean",
-    "contracts:run:nft": "cd ./contracts/ubiquistick && hardhat node",
-    "contracts:run:faucet": "cd ./contracts/dollar && yarn node:faucet",
-    "killscript": "kill `ps | grep dev-chain-init.sh | grep -v grep | awk '{ print $1 }'`",
-    "__format:cspell": "cspell '**'"
-  },
+  "name": "@ubiquity/monorepo",
+  "version": "1.0.1-beta",
```

Comment:
Would be dope to follow semver as part of our CI.

Reply:
nice catch!

Reply:
> nice catch! but cspell doesn't work properly for some variable names. ie. curveuADBalance fails on spell check. My suggestion is to make an issue if you actually want spell check with ignore words / paths and do it later

#252